docs/maps/search.asciidoc

```diff
@@ -2,8 +2,9 @@
 [[maps-search]]
 == Searching your data
 
-**Elastic Maps** embeds the query bar for real-time ad hoc search.
+**Elastic Maps** embeds the search bar for real-time search.
 Only layers requesting data from {es} are filtered when you submit a search request.
+Layers narrowed by the search context contain the icon image:maps/images/filter_icon.png[] to the right of layer name in the legend.
```

Comment:
Can we give this icon a name?  So it reads something like "contain the tool icon"